tools/FakeItEasy.Build/Program.cs

```diff
@@ -80,8 +81,8 @@ public static void Main(string[] args)
                 Target(
                     testSuite.Key,
                     DependsOn("build", "testsDirectory"),
-                    forEach: testSuite.Value,
-                    action: testDirectory => RunTests(testDirectory));
+                    forEach: testSuite.Value.SelectMany(project => GetFrameworks(project).Select(framework => new { Project = project, Framework = framework })),

```

Comment:
Maybe use a tuple here? The output would be more compact (`(tests/FakeItEasy.Tests, netcoreapp1.0)` instead of `{ Project = tests/FakeItEasy.Tests, Framework = netcoreapp1.0 }`. Also, it would let you extract the whole expression to a `GetProjectFrameworkPairs` method, for better readability.